tasks/section5.yml

```diff
@@ -171,15 +270,20 @@
       - rule_5.2.1
 
 - name: "SCORED | 5.2.1 | PATCH | Ensure permissions on /etc/ssh/sshd_config are configured"
-  command: /bin/true
+  file:
+      dest=/etc/ssh/sshd_config
+      state=file
+      owner=root
+      group=root
+      mode=0600
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md